README.md

```diff
@@ -40,6 +31,20 @@ Learn more about [Working Groups](Working-Groups.md).
 * None yet.
 
 
+## The Planetary Software Organization Technical Steering Committee
+
+The Technical Steering Committee (TSC) is the governing body of the
+Planetary Software Organization, as described in its
+[Charter](TSC-Charter.md).
+
+The TSC primarily provides responsive input and organizational help
```

Comment:
What about - 'The TSC provides responsive input and organizational help to Top-Level Projects and Working Groups, as well as consultation by other interested software projects or endeavors.'